packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -525,91 +524,6 @@ export class SDKClient {
     }
   };
 
-  async executeGetTransactionRecord(
```

Comment:
Don't remove this.
I think it's not called because of a bug and in follow up we should get logic using this method instead of repeating limiter and metrics calls everywhere.
SHould sync with @georgi-l95 PR to see incoming logic and after both we revisit again for conformity

Reply:
Reverted this change

Reply:
So `executeGetTransactionRecord()` is the function that get transaction records using the SDK via `getRecord()` method. And like we've discussed before, `getRecord()` cost operator some fees that don't get captured in metrics and rate limitter class. That's why we wanted to replace `getRecord()` with calls to mirror-node like this [issue#2706](https://github.com/hashgraph/hedera-json-rpc-relay/issues/2706) is trying to solve. I'm working on a solution for this ticket, and the solution makes repeated calls to mirror-node to get the correct `transactionRecord` using `transactionId`. Now this solution is expected to do the exact same job as the `executeGetTransactionRecord`, so in my opinion, this `executeGetTransactionRecord()` can be removed safely.

Reply:
@quiet-node agreed in the final case but let's do the SDK `getRecord()` replacement in follow up for `executeGetTransactionRecord()`.
Avoid conflict with PRs currently out also